tests/TestCase/Validation/ValidatorTest.php

```diff
@@ -630,14 +630,14 @@ public function testAllowEmptyFor()
         $results = $validator->errors(['name' => null]);
         $this->assertSame([], $results);
 
-        $this->assertSame([], $results);
         $results = $validator->errors(['name' => '']);
-
         $this->assertSame([], $results);
-        $results = $validator->errors(['title' => 0]);
 
+        $results = $validator->errors(['title' => 0]);
```

Comment:
Did you mean 'name' here?

Reply:
Doh, I have messed the field names in both PR. Thanks for point it out.

Reply:
I missed it too and merged it :(